packages/react-utilities/src/hooks/useEventCallback.ts

```diff
@@ -5,11 +5,11 @@ import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect';
  * https://reactjs.org/docs/hooks-faq.html#how-to-read-an-often-changing-value-from-usecallback
  *
  * Modified `useCallback` that can be used when dependencies change too frequently. Can occur when
- * e.g. user props are depedencies which could change on every render
+ * e.g. user props are dependencies which could change on every render
  * e.g. volatile values (i.e. useState/useDispatch) are dependencies which could change frequently
  *
  * This should not be used often, but can be a useful re-render optimization since the callback is a ref and
- * will not be invalidated between rerenders
+ * will not be invalidated between re-renders
```

Comment:
That's actually an interesting change 😀 Even React does are not consistent in "re-render" vs "rerender":

![image](https://user-images.githubusercontent.com/14183168/128008784-6e1d355a-3fe7-4994-8ed5-b7e845126604.png)
![image](https://user-images.githubusercontent.com/14183168/128008695-5eea530e-f12e-4098-8195-dc6b67599317.png)

There was even a poll on Twitter about it, https://twitter.com/kentcdodds/status/1290424999149334528